src/components/AddWorkout.jsx

```diff
@@ -0,0 +1,116 @@
+import { useEffect } from "react";
+import axios from "axios";
+import { useState } from "react";
+import { Paper } from "@mui/material";
+import { paperStyles } from "../styles/styles";
+import AddIcon from '@mui/icons-material/Add';
+import { IconButton } from "@mui/material";
+import Dialog from '@mui/material/Dialog';
+import DialogTitle from '@mui/material/DialogTitle';
+import DialogContent from '@mui/material/DialogContent';
+import DialogActions from '@mui/material/DialogActions';
+import CloseIcon from '@mui/icons-material/Close';
+import { styled } from '@mui/material/styles';
+import Typography from '@mui/material/Typography';
+import Button from '@mui/material/Button';
+
+
+
+
+function AddWorkout() {
+// workout is an array of exercises
+  const [workout, setWorkout] = useState([]);
+  const [dialogOpen, setDialogOpen] = useState(false);
+
+    let data, filterMethod = "muscle", filterType = "biceps" 
+
+    useEffect(() => {
+        axios.get('https://api.api-ninjas.com/v1/exercises'+`?${filterMethod}=${filterType}`, {headers: { 'X-Api-Key': '/2l4M9V5VpBLETtKJzc5Mw==mPTtJXrI5qzXquuf' }})
```

Comment:
the url and the key is in a variable in api.js, we should use it because we might have to do multiple api call

Reply:
yes, i will do that